src/app/core/auth_provider.js

```diff
@@ -29,7 +30,11 @@
     }
 
     function me() {
+      if(_me)
+        return $q.when(_me);
+
       return $http.get(`${ENV.HOST}/me`)
```

Comment:
This is returning true/false instead of an actual user. I wonder if this is my problem or if the API is actually returning true/false instead of the currentUser.